tests/testthat/test_special_orthogonal_group.R

```diff
@@ -95,3 +95,43 @@ test_that("Tests Vector From Skew Matrix", {
   vec.from.skew.matrix <- so3$VectorFromSkewMatrix(skew.mat = skew.mat)
   expect_equivalent(vec, vec.from.skew.matrix)
 })
+
+test_that("Tests Vector From Skew Matrix", {
+  so3 <- SpecialOrthogonalGroup$new(n = 3)
+  rot.mat <- array(c(1, 0, 0,
+                     0, cos(.12), sin(.12),
+                     0, -sin(.12), cos(.12)), dim = c(3, 3))
+  rot.vec <- so3$RotationVectorFromMatrix(rot.mat)
+  expected <- ToNdarray(array(c(.12, 0, 0)), to.ndim = 2)
```

Comment:
There's a sign problem here. expected should be (-0.12, 0., 0.).

See rotation matrix of axis x:
https://en.wikipedia.org/wiki/Rotation_matrix

Reply:
So this was confusing me at first as well -- and I think the way I have the code written makes it very confusing.  The array function populates a matrix by columns and not rows.   So the rotation matrix is actually 
     [,1]      [,2]       [,3]
[1,]    1 0.0000000  0.0000000
[2,]    0 0.9928086 -0.1197122
[3,]    0 0.1197122  0.9928086

Maybe I will try to use rbind() to make it more readable and intuitive in the future

Reply:
Ah I see. Yes, any chance you could change it? thanks!